services/proxy/README.md

```diff
@@ -128,6 +128,14 @@ somewhat costly operation, especially if the user is a member of a large number
 groups. If the group memberships of a user are changed in the IDP after the
 first login, it can take up to 5 minutes until the changes are reflected in Infinite Scale.
 
+### Claim Updates
+
+OpenID Connect (OIDC) scopes are used by an application during authentication to authorize access to a user's detail, like name or picture. Each scope returns a set of user attributes, which are called claims. The scopes an application should request, depends on which user attributes the application needs. Once the user authorizes the requested scopes, the claims are returned in an ID Token.
+
+Claims cant get updated automatically in Infinite Scale when there is a change in the IDM (identity management). JWT tokens generated by the IDP (OpenID Connect provider) requested by Infinite Scale, are immutable, means they reflect the IDM claim state when issued. Therefore you can't change or update claims on existing tokens. To provide updated claim information for users in Infinite Scale, a new immutable token must be requested. This especially affects claim changes like changed user details or group names and applies for users individually.
```

Comment:
I guess the main purpose of the "Claim Updates" section is to document the fact that with autoprovsioning enabled updates to user attributes/group membership on IDP side of things (please don't call it IDM here, as that will cause confusion with the `idm` service in ocis) are not immediately reflected on the oCIS side.

There are multiple reasons for that:

1. There is no active sync process between the IDP and oCIS, the autoprovisioning works based on the  users claims returned by the IDP's userinfo endpoint (or extracted from the access token when  `PROXY_OIDC_SKIP_USER_INFO` is true)
2. If e.g. the group membership of a user is changed in the IDP, the earliest possible time that ocis will notice it is, when the current access token expired and a new access token is issued by the IDP.
3. There might be an additional delay if the last change of groupmemberships was less than 5 minutes ago. (This was introduced as the whole process of  checking and updating group memberships is somewhat resource intensive)

I guess we should also document the facts  that
-  oCIS can't differentiate between a group being renamed in the IDP and a users being reassigned to a different group. 
- oCIS does not get aware when a groups is being deleted in the IDP

Reply:
For completness, after chatting:
Group renaming or deletion has an impact on sharing or space membership.
ocis cant distinguish between the cases. Users will get access rejected.